scriptExecutor.go

```diff
@@ -271,3 +274,36 @@ func buildDockerRunCommand(executionConf *executionConf) (string, error) {
 	return finalScript, nil
 
 }
+
+// Writes input vars to env file
+func writeToEnvFile(envMap map[string]string, filename string) error {
+	content, err := marshal(envMap)
+	if err != nil {
+		return err
```

Comment:
since we control this logic now, can we add proper error messages here? Applies at other places as well.

Reply:
done